doc/api/modules.md

```diff
@@ -791,4 +800,5 @@ be explicitly exported in order to be used.
 [`path.dirname()`]: path.html#path_path_dirname_path
 [exports shortcut]: #modules_exports_shortcut
 [module resolution]: #modules_all_together
-[native addons]: addons.html
\ No newline at end of file
+[native addons]: addons.html
```

Comment:
cc @nodejs/documentation 
This diff appears because we have not line break in this last line previously. Maybe it is worth to set some rule for this when doc linting is landed.